### PR TITLE
Bump version to v0.2.2

### PR DESCRIPTION
**Pull request by waynr**
Includes:

* a cherry-pick of #156: Set a custom user agent for the godo client.
* some minor fixes to the Makefile

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-## unreleased
+## unreleased\n
```
Comment (nanzhong) [inline CHANGELOG.md]: extra `\n` here?

File: Makefile [hunk 2]
```diff
@@ -51,7 +50,7 @@ build:
 
 .PHONY: push
 push:
-ifeq ($(shell [[ $(BRANCH) != "master" && $(VERSION) != "dev" ]] && echo true ),true)
+ifeq ($(shell [[ $(BRANCH) != "release-0.2.0" && $(VERSION) != "dev" ]] && echo true ),true)
```
Comment (nanzhong) [inline Makefile]: Is this branch name change intentional?

Comment (waynr) [reply Makefile]: This was intentional, and is necessary for `make publish` to run on this branch. It's not a very good pre-release check to begin with, but this is a quick fix that unblocks publishing images for now. I've made a note to follow up con-1881 with a jira ticket to improve the pre-release checks on this repo (see description of #170)

File: Makefile [hunk 3]
```diff
@@ -29,7 +29,7 @@ bump-version:
 	@sed -i'' -e 's/${VERSION}/${NEW_VERSION}/g' README.md
 	$(eval NEW_DATE = $(shell date +%Y.%m.%d))
 	@sed -i'' -e 's/## unreleased/## ${NEW_VERSION} - ${NEW_DATE}/g' CHANGELOG.md
-	@ echo '## unreleased\n' | cat - CHANGELOG.md > temp && mv temp CHANGELOG.md
+	@ echo "## unreleased\n" | cat - CHANGELOG.md > temp && mv temp CHANGELOG.md
```
Comment (jcodybaker) [inline Makefile]: I think you actually want printf instead of echo.
```
$ echo "hello\n"
hello\n
```

Comment (waynr) [reply Makefile]: d'oh...this is what I get for using `zsh` as my default shell

**Comment by waynr** [comment]
@nanzhong @jcodybaker I've addressed your comments, thanks!

**Comment by timoreimann** [comment]
Should we still try to get this merged / released?

**Comment by timoreimann** [comment]
Closing this one as we stopped supporting the 0.2.x branch. See also #261.

**Pull request closed without merging**